### ISSUE TITLE
in a Quora answer, the "more" link doesn't show up in link hints

### ISSUE THREAD

**Issue by rwang2022** [philc/vimium#4539]
**Describe the bug**
If you go to any quora answer, usually there is a blue "more" link to expand the answer. This doesn't show up in link hints when you press f. 

![image](https://github.com/user-attachments/assets/4f04ecd7-46dd-4d22-8399-ad1809b39732)
The jk hint is for the 3 dots. There is no link hint for the blue "more"

**Browser and Vimium version**

This is Vimium 2.1.2 on Google Chrome 128.0.6613.113 (Official Build) (64-bit). I'm on Linux.

**Comment by UncleSnail** [comment]
Thank you for the bug report.

This issue is because Quora does not label that link correctly. It is a div that is only clickable because of JS interactions. There is no indication in the HTML that the element is clickable, so other software (like screen readers) that relies on HTML will also not recognize this as clickable. Vimium does not support or try to accommodate incorrect uses of HTML elements and it would be impractical to try to do so. If the website owner correctly labels an element as clickable, it will get a link hint. If not, it will not.

Unfortunately, there is nothing that can be done about this issue except telling Quora to fix their website.

**Comment by nanoraptor** [comment]
True. It is a bad way to present the website from Quora's end while all the other major sites clearly mention the clickable link in HTML code.
